scripts/nb-tester/test-notebook.py

```diff
@@ -14,78 +14,149 @@
 from pathlib import Path
 import nbconvert
 import nbformat
+import nbclient
```

Comment:
Total nit: Python convention is to split first-party and third-party library imports. Start with first-party, then blank line, then third-party.

(In a Python repo, I would have set up isort to automate this.)